cdap-cli/src/main/java/co/cask/cdap/cli/util/table/AltStyleTableRenderer.java

```diff
@@ -69,7 +69,7 @@
  */
 public class AltStyleTableRenderer implements TableRenderer {
 
-  private static final int DEFAULT_WIDTH = 80;
+  public static final int DEFAULT_WIDTH = 80;
```

Comment:
Actually you should use this constants in the tests. And it can be package visible for the test with @VisibleForTesting